Installing-Oppia-using-Docker.md

```diff
@@ -184,5 +184,46 @@ The Oppia development environment provides additional `make` commands that you c
 - `make init`: Initializes the Oppia development environment by building the Docker Images and starting the dev-server.
 - `make echo_flags`: This command shows the flags with thier values that are being used by the Oppia development server.
 
+## Running tests
+
+Once you have Oppia installed using Docker, you can execute a variety of tests to ensure the functionality and quality of the code. These tests check various parts of the application, such as code style (linting), server-side functionality (backend), client-side functionality (frontend), and how the application behaves in different scenarios (acceptance testing).
+
+## Available make commands for running tests
+
+The following table lists the available `make` commands for running tests:
+
+| Command                                                    | Description                                                      | Purpose                                                                           |
+| ---------------------------------------------------------- | ---------------------------------------------------------------- | ------------------------------------------------------------------------------------------- |
+| `make run_tests.lint`                                        | Runs the linter tests.                                           | Checks code style and formatting for consistency.                                           |
+| `make run_tests.backend`                                   | Runs the backend tests.                                          | Verifies the functionality of backend logic and database interactions.                      |
+| `make run_tests.frontend`                                    | Runs the frontend unit tests.                                    | Tests individual components and functions of the frontend code.                             |
```

Comment:
These commands have flags too.

https://github.com/oppia/oppia/blob/6171792008266ad07826564316712b4b7b10eb61/Makefile#L103

ex:
```
make run_tests.frontend PYTHON_ARGS="--flag1 --flag2"
```

The flags are from here: https://github.com/oppia/oppia/blob/6171792008266ad07826564316712b4b7b10eb61/scripts/run_frontend_tests.py#L42-L79

Reply:
Since, no separate test files exist for `make run_tests.lighthouse_accessibility` and `make run_tests.lighthouse_performance` but a single ![file](https://github.com/oppia/oppia/blob/develop/scripts/run_lighthouse_tests.py#L57-L79). Should I assume both of them share the same PYTHON_ARG flags?

Also, I shifted from tabular form to bullets to mention flags along with respective make commands. Can you have a look once @gp201?